test/suites/container_devices_nic_bridged.sh

```diff
@@ -36,6 +36,17 @@ test_container_devices_nic_bridged() {
   # Record how many nics we started with.
   startNicCount=$(find /sys/class/net | wc -l)
 
+  # Test device renaming without "eth0" device applied by profile.
+  lxc profile copy default "${ctName}"
+  lxc profile device remove "${ctName}" eth0
+  lxc init testimage "${ctName}" -p "${ctName}"
+  lxc config device add "${ctName}" eth0 nic network=${brName}
+  lxc config show -e "${ctName}" | sed '/^devices:/,/type:/ s/eth0/eth1/' | lxc config edit "${ctName}" # Rename added "eth0" to "eth1"
+  lxc config show -e "${ctName}" | grep -q "eth1"
```

Comment:
Since `lxc config show -e` include the volatile keys that contain the NIC name it would be safer to directly get the device key you expect to be there (or absent):

```suggestion
  ! lxc config device get "${ctName}" eth0 name || false
  [ "$(lxc config device get "${ctName}" eth1 name)" = "eth1" ]
  [ "$(lxc config device get "${ctName}" eth1 network)" = "${brName}" ]
```